src/content/extensions/multipleroutes/src/utils.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var pn = chrome.privacy.network;
+var pi = null;
+
+// Handle the case when this is installed in pre-M48.
+if (pn.webRTCIPHandlingPolicy === undefined) {
+  chrome.privacy.IPHandlingPolicy = {};
+  chrome.privacy.IPHandlingPolicy.DEFAULT = 0;
+  chrome.privacy.IPHandlingPolicy.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES = 1;
+  chrome.privacy.IPHandlingPolicy.DEFAULT_PUBLIC_INTERFACE_ONLY = 2;
+  chrome.privacy.IPHandlingPolicy.DISABLE_NON_PROXIED_UDP = 3;
+}
+
+pi = chrome.privacy.IPHandlingPolicy;
+
+// Helper function to convert the parameters to policy synchronously.
+function convertToPolicy(allowMultiRoute, allowUdp, isInstall) {
+  if (allowMultiRoute) {
+    if (isInstall && pn.webRTCIPHandlingPolicy !== undefined) {
+      // If we're installing the extension, we'll default to a more private
+      // mode.
+      return pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES;
```

Comment:
Why does the output of the convert function check pn.webRTCIPHandlingPolicy?  Shouldn't the convert function always return the same thing regardless of pn.webRTCIPHandlingPolicy?